cider/tests/nextgen/jitlib/JITLibTest.cpp

```diff
@@ -141,6 +174,112 @@ TEST_F(JITLibTests, LogicalOpTest) {
       true, false, false, [](JITValue& a, JITValue& b) { return !a; });
   executeBinaryOp<JITTypeTag::BOOL>(
       false, true, true, [](JITValue& a, JITValue& b) { return !a; });
+
+  // and
+  executeBinaryOp<JITTypeTag::BOOL>(
+      true, false, false, [](JITValue& a, JITValue& b) { return a && b && true; });
```

Comment:
Q: why we need extra ```&& true``` as part of return? Same goes for others.

Reply:
> Q: why we need extra `&& true` as part of return? Same goes for others.

OK, that's test for JITValue operator constant value in some cases，like 'a&&true, b||false', same as the 'a+b+1' above

Reply:
I c. Thanks for clarification.